pages/common/nest.md

```diff
@@ -0,0 +1,24 @@
+# nest
+
+> The Nest CLI is a command-line interface tool that helps you to initialize, develop, and maintain your NestJS applications.
```

Comment:
We try to avoid personal pronouns where possible.
```suggestion
> The Nest CLI is a command-line interface tool that helps to initialize, develop, and maintain NestJS applications.
```